tests/pages/components/user-sidebar.js

```diff
@@ -0,0 +1,15 @@
+import { isVisible, text, isHidden, property } from 'ember-cli-page-object';
+
+export default {
+  scope : '.user-sidebar',
+
+  userSidebarCount    : isVisible(),
+  name                : text('h2 .name'),
+  username            : text('h2 .username'),
+  twitterHandle       : text('li.twitter'),
+  website             : text('li.website'),
+  twitterHandleHidden : isHidden('li.twitter'),
+  websiteHidden       : isHidden('li.website'),
+  twitterLink         : property('href', 'li.twitter a'),
+  websiteLink         : property('href', 'li.website a')
```

Comment:
There are arguments for this type of formatting/spacing, but we don't use it anywhere in the app ourselves, so we would prefer if you switched this to just using a single space instead of horizontally aligning the values.